plasma_framework/contracts/mocks/exits/payment/routers/PaymentStandardExitRouterMock.sol

```diff
@@ -43,4 +47,20 @@ contract PaymentStandardExitRouterMock is PaymentStandardExitRouter {
     }
 
     function depositFundForTest() public payable {}
+
+    /**
+     * This function helps test reentrant by making this function itself the first call with 'nonReentrant' protection
+     * So all other PaymentExitGame functions that is protected by 'nonReentrant' too would fail as it would be considered as re-entrancy
```

Comment:
So all other PaymentExitGame functions that are protected by 'nonReentrant' too would fail as it would be considered as re-entrancy